src/exambank/exambank.py

```diff
@@ -0,0 +1,112 @@
+from io import BytesIO
+from pathlib import Path
+
+from pymupdf import Matrix
+from pymupdf import open as p_open
+from pypdf import PdfReader, PdfWriter
+from reportlab.lib import colors
+from reportlab.lib.pagesizes import A4
+from reportlab.lib.units import inch
+from reportlab.pdfgen import canvas
+
+# param to describe margin for exam generation text
+border = 20
+
+def create_watermark(
+        computing_id: str,
+        density: int = 5
+) -> BytesIO:
```

Comment:
can you add a docstring saying that the bytes object returned is a PDF with a single page, containing the watermark image? (if I understood correctly)

Additionally, would it make sense to return an image only, and not a pdf containing an image?

Reply:
Sure, resolved in 734f92a.

We could do that I think, that would require us to turn it into an image first though. Doing this might make it easier to replace pypdf with pymupdf later on.